src/literals.rs

```diff
@@ -15,8 +15,13 @@ use memchr::{memchr, memchr2, memchr3};
 use syntax;
 
 use freqs::BYTE_FREQUENCIES;
+
+#[cfg(feature = "simd-accel")]
```

Comment:
Yeah, that should work. Defining the shim in `lib.rs` is far better than using cfg here. I'll change this.